jasminewd2/index.d.ts

```diff
@@ -44,6 +44,13 @@ declare namespace jasmine {
     toThrowError(expected?: new (...args: any[]) => Error | Promise<new (...args: any[]) => Error>, message?: string | RegExp | Promise<string | RegExp>): Promise<void>;
   }
 
+  interface ArrayLikeMatchers<T extends ArrayLike<T>> extends Matchers<T> {
```

Comment:
@minestarks @LOZORD 
Seems like I misunderstood the initial question, I thought it was about the syntax used...

I used `ArrayLike` instead of `Array` because it was used in  `jasmine.d.ts` like that.

See https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/jasmine/index.d.ts#L75

I have no particular knowledge of the `jasminewd2/index.d.ts` cited here as I just applied the same changes as I did in the *base* jasmine typings

`ArrayLike` seems to match well with regular arrays like `[1,2,3,]` as you can se in these tests: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/14875/files#diff-443087a7bbdcb27d0da21ebf1b396e9fR108

```
const arr:ArrayLike<number> = [1, 2, 3];  // seems to be valid
```